watertap/tools/oli_api/client.py

```diff
@@ -267,7 +266,8 @@ def get_flash_history(self, dbs_file_id, job_id=False):
         :param dbs_file_id: string ID of DBS file
         :param job_id: bool flag to return job ID for troubleshooting
 
-        :return flash_history: dictionary containing array of submitted jobs OR just job ID if specified
+        :return flash_history: dictionary containing array of submitted jobs OR
+        just job ID if specified
```

Comment:
```suggestion
        :return flash_history: dictionary containing array of submitted jobs OR
            just job ID if specified
```
I think this change will resolve the RTD error, per https://stackoverflow.com/a/74798788